src/transformers/models/bigscience176b/configuration_bigscience176b.py

```diff
@@ -114,11 +114,17 @@ def __init__(
         masked_softmax_fusion=True,
         layer_norm_epsilon=1e-5,  # TODO
         initializer_range=0.02,  # TODO
-        use_cache=True,  # TODO
+        use_cache=False,  # TODO
         bos_token_id=0, # 50256,  # TODO
         eos_token_id=0, # =50256,  # TODO
+        apply_residual_connection_post_layernorm=False,
+        bias_dropout_fusion=True,
+        skip_bias_add=True,
+        hidden_dropout=0.1,
+        attention_dropout=0.1,
```

Comment:
Can you check all the arguments are in the docstring?

Reply:
Should be resolved [here](https://github.com/thomwolf/transformers/pull/1/commits/6b22a495f20f6d1ead8ac6f1b8d8002d7c7ac4a5)